ext/MitsubaLoader/CMitsubaScene.cpp

```diff
@@ -49,58 +48,17 @@ bool CMitsubaScene::processChildData(IElement* _child)
 			return false;
 		}
 
-		const asset::ICPUMesh* shapeMesh = shape->getMesh();
-		return appendMesh(shapeMesh, shape->getTransformMatrix());
+		const core::smart_refctd_ptr<asset::ICPUMesh> shapeMesh = shape->getMesh();
```

Comment:
what are you doing here?

Reply:
here i retrive mesh from shape element

Reply:
then this retrived mesh is added to the array which contain all loaded meshes

Reply:
I meant ref-counting-wise?